js_modules/dagit/packages/core/src/runs/LogsRowStructuredContent.tsx

```diff
@@ -358,6 +358,48 @@ const FailureContent: React.FC<{
   );
 };
 
+const StepUpForRetryContent: React.FC<{
+  message?: string;
+  error?: PythonErrorFragment | null;
+}> = ({message, error}) => {
+  let contextMessage = null;
+  let errorCause = null;
+
+  if (message) {
+    contextMessage = (
+      <>
+        <span>{message}</span>
+        <br />
+      </>
+    );
+  }
+
+  if (error && error.cause) {
```

Comment:
in the case of direct retry request being thrown from op can you still click on the event to see the full dialog?

Reply:
You cannot, it looks like it used to. Is there value in seeing that stack trace? It's entirely within Dagster code

![Screen Shot 2022-05-02 at 5 26 23 PM](https://user-images.githubusercontent.com/22018973/166490429-1af7ea68-0938-4f5c-9b14-6db6991ffdcc.png)

Reply:
> it's entirely within Dagster code

the bottom of the stack is in the op fn, which may be meatier if a user had deeper fn nesting in their op

Its unlikely worth showing in the collapsed form - but I think you should be able to click on it to see the full stack trace, just like the `cause` one shows both the `cause` and the retry request stack in the full dialog

Reply:
@hellendag is it possible to hide the trace in the collapsed form but still show the 'View full message' option?

Reply:
if i recall correctly the "view full message" thing only procs when the collapsed form thinks its rendering got truncated instead of some set-up where it knows it has more information to display, so it might be tough to achieve the improved behavior. 

maybe @hellendag has an idea here - or its something to file away for a follow-up, since I believe there are cases where we don't allow seeing the full exception chain on other error events if you have a very short stack trace and it doesn't proc the height based expansion dialog thing.

Reply:
You don't have to rely on the cell truncation detection. I think you could just render something like this, within the rendered error content:

```
<ButtonLink
  color={Colors.Dark}
  underline="always"
  onClick={() => showCustomAlert({body: <PythonErrorInfo error={error} />})}
>
  Show error
</ButtonLink>
```